src/EditorFeatures/CSharpTest/Diagnostics/GenerateMethod/GenerateMethodTests.cs

```diff
@@ -2879,84 +2880,31 @@ private static void OnChanged(object sender, EventArgs e)
 }");
         }
 
-        public class GenerateConversionTest : AbstractCSharpDiagnosticProviderBasedUserDiagnosticTest
```

Comment:
This was a nested class which accidentally caused me grief. I moved it to a separate file.